nonsemantic/NonSemantic.Shader.DebugInfo.200.asciidoc

```diff
@@ -0,0 +1,815 @@
+SPIR-V NonSemantic Shader DebugInfo Instructions
+================================================
+
+:result_type: pass:normal['Result Type' must be *OpTypeVoid*.]
+:source:      pass:normal['Source' is a *DebugSource* instruction representing the text of the source program]
+:flags:       pass:normal['Flags' is the '<id>' of a 32-bit integer *OpConstant* formed by the bitwise-OR of values from the <<DebugFlags,*Debug Info Flags*>> table.]
+:NonSemantic100URL: https://github.com/KhronosGroup/SPIRV-Registry/blob/main/nonsemantic/NonSemantic.Shader.DebugInfo.100.asciidoc
+
+Contact
+-------
+
+To report problems with this extension, please open a new issue at:
+
+https://github.com/KhronosGroup/SPIRV-Headers
+
+Contributors and Acknowledgments
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+ - Alexey Sotkin, Intel
+ - Dmitry Sidorov, Intel
+ - Mariya Podchishchaeva, Intel
+ - Nikita Kornev, Intel
+ - Stanley Gambarin, Intel
+
+Author of *NonSemantic.Shader.DebugInfo.100* specification.
+
+ - Baldur Karlsson, Valve
+
+Contributors to original *OpenCL.DebugInfo.100* specification.
+
+ - Yaxun Liu, AMD
+ - Brian Sumner, AMD
+ - Ben Ashbaugh, Intel
+ - Alexey Bader, Intel
+ - Raun Krisch, Intel
+ - Pratik Ashar, Intel
+ - John Kessenich, Google
+ - David Neto, Google
+ - Neil Henning, Codeplay
+ - Kerch Holt, Nvidia
+ - Jaebaek Seo, Google
```

Comment:
I would not re-list the contributors to other specs by name. (similar to how we don't re-list the instructions in the .100 specs).